### PR TITLE
Created DagbladetCrawlerBase and updated relevant crawlers

### PR DESCRIPTION
**Pull request by datagutten**

### CODE REVIEW

File: comics/aggregator/crawler.py [hunk 1]
```diff
@@ -250,3 +250,25 @@ def crawl_helper(self, short_name, pub_date):
...
+
+
+class DagbladetCrawlerBase(CrawlerBase):
+    """Base comics crawler for all comics posted at dagbladet.no"""
+    headers = {'User-Agent': 'Mozilla/5.0'}
+    time_zone = 'Europe/Oslo'
+
+    def crawl_helper(self, short_name, pub_date):
+        epoch = self.date_to_epoch(pub_date)
+        page_url = 'http://www.dagbladet.no/tegneserie/%s' % (short_name)
```
Comment (jodal) [inline comics/aggregator/crawler.py]: The parenthesis here have no effect. If you want to make it a tuple, add a trailing comma.

File: comics/aggregator/crawler.py [hunk 2]
```diff
@@ -250,3 +250,25 @@ def crawl_helper(self, short_name, pub_date):
...
+        page = self.parse_page(page_url)
+        # Find the strip for the wanted date
+        date_string = pub_date.strftime('%Y-%m-%dT00:00:00+02:00')
+        time = page.root.xpath('//time[@datetime="%s"]' % (date_string))
+        # No release that date
+        if not time:
+            return
+        # Get the image
+        article = time[0].getparent().getparent()
+        image = article.find('.//img').get('src')
```
Comment (jodal) [inline comics/aggregator/crawler.py]: I think this works:

``` python
url = article.src('.strip-container img')
return CrawlerImage(url)
```

This also keeps to the naming convention or the image URL variable and avoids selecting any images from the social buttons row.

File: comics/aggregator/crawler.py [hunk 3]
```diff
@@ -250,3 +250,25 @@ def crawl_helper(self, short_name, pub_date):
...
+        epoch = self.date_to_epoch(pub_date)
+        page_url = 'http://www.dagbladet.no/tegneserie/%s' % (short_name)
+        page = self.parse_page(page_url)
+        # Find the strip for the wanted date
+        date_string = pub_date.strftime('%Y-%m-%dT00:00:00+02:00')
+        time = page.root.xpath('//time[@datetime="%s"]' % (date_string))
+        # No release that date
+        if not time:
+            return
+        # Get the image
```
Comment (jodal) [inline comics/aggregator/crawler.py]: I think you can drop all the comments without loosing any meaning. Maybe use a couple of blank lines to group code blocks instead.

File: comics/aggregator/crawler.py [hunk 4]
```diff
@@ -250,3 +250,25 @@ def crawl_helper(self, short_name, pub_date):
...
+    headers = {'User-Agent': 'Mozilla/5.0'}
+    time_zone = 'Europe/Oslo'
+
+    def crawl_helper(self, short_name, pub_date):
+        epoch = self.date_to_epoch(pub_date)
+        page_url = 'http://www.dagbladet.no/tegneserie/%s' % (short_name)
+        page = self.parse_page(page_url)
+        # Find the strip for the wanted date
+        date_string = pub_date.strftime('%Y-%m-%dT00:00:00+02:00')
+        time = page.root.xpath('//time[@datetime="%s"]' % (date_string))
```
Comment (jodal) [inline comics/aggregator/crawler.py]: Same here, the parenthesis has no effect as they lack the trailing comma making it a one-tuple.

File: comics/comics/fagprat.py [hunk 5]
```diff
@@ -11,14 +11,10 @@ class ComicData(ComicDataBase):
     rights = 'Flu Hartberg'
 
 
-class Crawler(CrawlerBase):
+class Crawler(DagbladetCrawlerBase):
     history_capable_date = '2010-11-15'
```
Comment (jodal) [inline comics/comics/fagprat.py]: Should be `history_capable_days = 14`.

File: comics/comics/firekanta.py [hunk 6]
```diff
@@ -11,14 +11,12 @@ class ComicData(ComicDataBase):
     rights = 'Nils Axle Kanten'
 
 
-class Crawler(CrawlerBase):
-    history_capable_days = 30
+class Crawler(DagbladetCrawlerBase):
+    # History capability is 6 releases,
+    # but it is only released three days per week
```
Comment (jodal) [inline comics/comics/firekanta.py]: Drop the comment :-)

File: comics/aggregator/crawler.py (hunk 2 above)
Comment (jodal) [inline comics/aggregator/crawler.py]: I might be off here, as article is an lxml object and not a comics object. If so, just change the variable name from `image` to `url`.

Comment (datagutten) [inline comics/aggregator/crawler.py]: Tried and got this: "AttributeError: 'HtmlElement' object has no attribute 'src'".
Will change the variable name.

**Comment by jodal** [comment]
Perfect! Thanks :-)

**Pull request merged**